pysignalscope/scope.py

```diff
@@ -853,7 +921,801 @@ def plot_channels(*channel: List['Scope'], timebase: str = 's', figure_size: Opt
         return fig
 
     @staticmethod
-    def scope2plot(csv_file, scope: str = 'tektronix', order: str = 'single', timebase: str = 's',
+    def check_limits(cur_value: float, min_value: float, max_value: float) -> bool:
+        """
+        Check if the current value is within the given range.
+
+        Example for a valid value:
+        >>> bool valid
+        >>> value = 10.2
+        >>> valid = HandleScope.check_limits(value, 3.2,11.3)
+        >>> if valid:
+        >>>     print(f"{value} is within the limit")
+        >>> else:
+        >>>     printf(f"{value} is invalid")
+        The current value will be check according the given limits.
+        If the value is within the limit the function provide True as return value.
+
+        :param cur_value: value to check
+        :type cur_value: float
+        :param min_value: lowest valid value
+        :type min_value: float
+        :param max_value: highest valid value
+        :type max_value: float
+
+        :return: condition result 'value is in range'
+        :rtype: bool
+        """
+        # Init return variable
+        ret_val = True
+
+        # Check for maximum
+        if cur_value > max_value:
+            ret_val = False
+        # Check for maximum
+        if cur_value < min_value:
+            ret_val = False
+
+        return ret_val
+
+    @staticmethod
+    def calculate_min_diff(cur_channel: np.array, ch_id: any) -> [bool, float]:
+        """
+        Check if the current value is within the given range.
```

Comment:
may we should use the word `actual` instead of `current`, as there is a danger of confusion with `current flowing in a circuit`

Reply:
I agree with this item.
prefix 'current' is change to 'actual' at the correspondent places.
Done